ui/raidboss/data/04-sb/alliance/orbonne_monastery.js

```diff
@@ -194,9 +188,7 @@ export default {
       netRegexJa: NetRegexes.ability({ id: '3851', source: '模造聖域' }),
       netRegexCn: NetRegexes.ability({ id: '3851', source: '仿制圣域' }),
       netRegexKo: NetRegexes.ability({ id: '3851', source: '모조 성역' }),
-      run: (data, matches) => {
-        data.halidom.push(matches.target);
-      },
+      run: (data, matches) => data.halidom.push(matches.target),
```

Comment:
Technically this will return the length of `data.halidom`?

Reply:
`run` is a function with a `void` return type, so I think this is fine.  Similarly, `T1 Initiated` is `run: (data) => data.started = true,` which returns `true` in my triggerset patch.